### DISCUSSION TITLE
FEplot now supports xdmf files visualization!

### DISCUSSION THREAD

**Discussion by ZAARAOUI999** [adtzlr/felupe#590] [Show and tell]
Hi @adtzlr I wish you well and all your family, I'm just going to tell you that we are able now to visualize all the data that are passed to the xdmf file. Here is an example:

```python
import felupe as fem
from feplot import Plotter 

mesh = fem.Cube(n=6)
region = fem.RegionHexahedron(mesh)
field = fem.FieldContainer([fem.Field(region, dim=3)])

boundaries, loadcase = fem.dof.uniaxial(field, clamped=True)

umat = fem.OgdenRoxburgh(material=fem.NeoHooke(mu=1), r=3, m=1, beta=0)
solid = fem.SolidBodyNearlyIncompressible(umat, field, bulk=5000)

move = fem.math.linsteps([0, 1, 0, 1, 2, 1], num=5)
step = fem.Step(items=[solid], ramp={boundaries["move"]: move}, boundaries=boundaries)

job = fem.CharacteristicCurve(steps=[step], boundary=boundaries["move"])
job.evaluate(filename="result.xdmf")

p = Plotter()
p.read_xdmf("result.xdmf")
```
Then you should get this;

https://github.com/adtzlr/felupe/assets/115699524/f1be716b-e3d1-433a-9e7b-7310959c053d

**Comment by adtzlr** [comment]
Hi @ZAARAOUI999, great news! Hope you are fine?

That means you switched from the matplotlib backend to pyvista and from static plots to an interactive app? 🚀 

Also, XDMF-support means FEplot now works with any time-series data, e.g. from dolphin (fenics)?

**Reply by ZAARAOUI999** [reply]
```That means you switched from the matplotlib backend to pyvista and from static plots to an interactive app?```
Exactly.
```with any time-series data```
I'm not totally sure, but this depends on the capability of the xdmf reader of ```meshio``` package.
